main.js

```diff
@@ -33,4 +34,156 @@ iconBars.addEventListener('click', openMenu);
 iconEx.addEventListener('click', closeMenu);
 link.forEach((x) => {
   x.addEventListener('click', closeMenu);
+});
+
+// Modal Functionalities
+const section = document.querySelector('#work .container');
+
+const modalList = [
```

Comment:
- [ ] [Optional] It would be better if you used ```name``` instead of ```title``` and ```technologies``` instead of ```languages``` for more descriptive names